src/MatrixToString.cpp

```diff
@@ -0,0 +1,68 @@
+#include <sstream>
+#include <string>
+
+#include "MatrixObject.h"
+
+int getWidth(const matrixObject (&matrix)[30][20]) {
+  int i = -1;
+  bool outOfRange = false;
+  while (i < 30 && !outOfRange) {
+    ++i;
+    if (!matrix[i][0].isValid) {
+      outOfRange = true;
+    }
+  }
+  return i;
+}
+
+int getHeight(const matrixObject (&matrix)[30][20]) {
+  int i = -1;
+  bool outOfRange = false;
+  while (i < 20 && !outOfRange) {
+    ++i;
+    if (!matrix[0][i].isValid) {
+      outOfRange = true;
+    }
+  }
+  return i;
+}
```

Comment:
What tries to achieve this function?
Similarly for the one above this.
@Ducktape10

Reply:
This function will be remove, because we get the size of map from the input.